presto-common/src/main/java/com/facebook/presto/common/block/Block.java

```diff
@@ -361,4 +361,14 @@ default Block getLoadedBlock()
      * The original block won't be modified.
      */
     Block appendNull();
+
+    /**
+     * Returns the converted long value at {@code position} if the value ar {@code position} can be converted to long.
+     *
+     * @throws UnsupportedOperationException if value at {@code position} is not compatible to be converted to long
+     */
+    default long toLong(int position)
```

Comment:
This needs more explanation in the comment. can you clarify how this is different from getLong() from the same class?